scripts/GetClrDbg.sh

```diff
@@ -369,39 +423,18 @@ else
 
     # For the rest of this script we can assume the working directory is the install path
 
-    echo 'Info: Determining Runtime ID'
-    __RuntimeID=
-    get_dotnet_runtime_id
     if [ -z $__RuntimeID ]; then
-        echo "Error: Unable to determine dotnet Runtime ID. Please make sure that dotnet is installed and the platform is supported. Look at https://www.microsoft.com/net/core for supported platforms."
-        exit 1
+        echo 'Info: Determining Runtime ID'
+        __RuntimeID=
+        get_dotnet_runtime_id
+        if [ -z $__RuntimeID ]; then
+            echo "Error: Unable to determine dotnet Runtime ID. Please make sure that dotnet is installed and the platform is supported. Look at https://www.microsoft.com/net/core for supported platforms."
```

Comment:
I might suggest changing this error message to suggest explicitly indicating which runtime id to use.